sdk/iOS/OoyalaSkinSDK/React/OOReactSkinModel.m

```diff
@@ -414,6 +429,26 @@ - (void)handleSwitchPrevious {
   [self.player previousVideo];
 }
 
+//OS: called only by OOReactSkinBridgeModuleMain
+- (void)setEmbedCode:(NSString *)embedCode {
+  
+#warning: old API. Remove when SDK version > 4.46.0_GA. If you need to use old API, uncomment -playForJustChangedItem in -bridgeCurrentItemChangedNotification: of 'OOSkinPlayerObserver'
```

Comment:
What exactly should be removed ?

Reply:
Commented example of using deprecated API: //[self.player setEmbedCode:embedCode]; 
I'm afraid It's can be helpful until we release first RC new API.